R/vdiagram-data.R

```diff
@@ -63,21 +79,15 @@ historyMeasurementsLabel <- function(reportObject) {
   }
 }
 
+#' Set default historic flags
```

Comment:
what's the difference between this and the result of `historyMeasurementsLabel`?